packages/playground/wordpress/src/mu-plugins/auto_login.php

```diff
@@ -106,7 +106,7 @@ function playground_auto_login() {
 	/**
 	 * Check if the request is for the login page.
 	 */
-	if (false === stripos( wp_login_url(), $_SERVER['SCRIPT_NAME'] )) {
+	if (empty($_SERVER['SCRIPT_NAME']) || false === stripos( wp_login_url(), $_SERVER['SCRIPT_NAME'] )) {
```

Comment:
Why would SCRIPT_NAME be empty?

Reply:
Why does it run even if we don't make an HTTP request? It seems like a bug. Silencing this warning only hides the problem. What's the root cause and what's best way of addressing it?

Reply:
I will take a look at why this is happening.